Scripts/interactable/NumberLogic/MemoryPanel.lua

```diff
@@ -200,4 +258,103 @@ function MemoryPanel.client_setUvValue(self, value)
 	if value > 255 then value = 255 end
 	if value == math.huge then value = 0 end
 	self.interactable:setUvFrameIndex(value)
+end
+
+function MemoryPanel.client_onTinker(self, character, lookAt)
+	if mp_deprecated_game_version or not lookAt or character:getLockingInteractable() then return end
+
+	local mem_gui = sm.gui.createGuiFromLayout("$CONTENT_DATA/Gui/Layouts/MemoryPanelGui.layout", false, { backgroundAlpha = 0.5 })
+
+	local parts = {}
+	for k, v in pairs(self.data) do
+		parts[#parts + 1] = k .. ':' .. v
```

Comment:
`pairs` has an unspecified (read: implementation defined^1) iteration order

consider something like

```suggestion
	local keys = {}
	for k, v in pairs(self.data) do
		if v ~= 0 or k == 0 then table.insert(keys, k) end
	end
	table.sort(keys)

	for _, k in ipairs(keys) do
		table.insert(parts, k .. ": " .. self.data[k])
```

which preserves key order and only includes non-zero keys that arent addr 0

[^1] due to the way we store keys, the data is a sparse array, so most of the keys will be stored in the hash part of the table, meaning the order is dependent on the internal hash mechanisms